scipy/signal/signaltools.py

```diff
@@ -416,6 +451,25 @@ def convolve(in1, in2, mode='full'):
         ``same``
            The output is the same size as `in1`, centered
            with respect to the 'full' output.
+    method : str {'auto', 'direct', 'fft'}, optional
+        A string indicating what method used to perform the convolution.
+
+        ``direct``
+           The convolution is determined directly from sums, the definition of
+           convolution.
+        ``fft``
+           The Fourier Transform is used to perform the convolution by calling
+           `fftconvolve`.
+        ``auto``
+           A rough estimate to see which convolution method is faster (the
+           Fourier transform or direct method) and that method is chosen
+           (default).
+
+    Notes
```

Comment:
Ah, got it. I figured that method wouldn't be changed very often and rationalized the change that way.


On Mar 9, 2016, 8:10 PM -0600, endolith<notifications@github.com>, wrote:
>  
> Inscipy/signal/signaltools.py(https://github.com/scipy/scipy/pull/5608#discussion_r55625155):
>  
> >@@ -416,6 +451,25 @@ def convolve(in1, in2, mode='full'):>``same``>The output is the same size as `in1`, centered>with respect to the 'full' output.>+  method : str {'auto', 'direct', 'fft'}, optional>+    A string indicating what method used to perform the convolution.>+>+    ``direct``>+      The convolution is determined directly from sums, the definition of>+      convolution.>+    ``fft``>+      The Fourier Transform is used to perform the convolution by calling>+      `fftconvolve`.>+    ``auto``>+      A rough estimate to see which convolution method is faster (the>+      Fourier transform or direct method) and that method is chosen>+      (default).>+>+  Notes
>  
> I just meant that the Notes section should be after See Also. I'm not sure that the Parameter section is long enough to break it out to an Other Parameters section.
>  
>  
> —
> Reply to this email directly orview it on GitHub(https://github.com/scipy/scipy/pull/5608/files#r55625155).
>  
>  
>